modules/tinymce/src/core/main/ts/EditorView.ts

```diff
@@ -45,24 +43,21 @@ const transpose = function (inline, elm, clientX, clientY) {
 };
 
 // Checks if the specified coordinate is within the visual content area excluding the scrollbars
-const isXYInContentArea = function (editor: Editor, clientX, clientY) {
+const isXYInContentArea = (editor: Editor, clientX: number, clientY: number): boolean => {
   const bodyElm = SugarElement.fromDom(editor.getBody());
   const targetElm = editor.inline ? bodyElm : Traverse.documentElement(bodyElm);
   const transposedPoint = transpose(editor.inline, targetElm, clientX, clientY);
 
   return isInsideElementContentArea(targetElm, transposedPoint.x, transposedPoint.y);
 };
 
-const fromDomSafe = function (node) {
-  return Optional.from(node).map(SugarElement.fromDom);
-};
+const fromDomSafe = <T extends Node>(node: T): Optional<SugarElement<T>> =>
```

Comment:
Should that be `node: T | null` or `node: T | undefined` or something? If we actually have a `T` (assuming typescript isn't lying to us) then there's no reason to get `Optional` involved and we could just call `SugarElement.fromDom` directly.

Reply:
Good point, I'm assuming this would likely be getting passed in as `null`, so I'll add that.